h/models/group.py

```diff
@@ -40,13 +41,16 @@ class GroupMembership(Base):
 
     id = sa.Column("id", sa.Integer, autoincrement=True, primary_key=True)
     user_id = sa.Column("user_id", sa.Integer, sa.ForeignKey("user.id"), nullable=False)
+    user = relationship("User", back_populates="memberships")
```

Comment:
Adding `GroupMembership.user` and `GroupMembership.group` relationships as well. These seem worth having just for the convenience, and they're also used by `factories.GroupMembership` (see below).